src/containers/shared/components/Transaction/XChainBridge.tsx

```diff
@@ -42,3 +42,7 @@ export const XChainBridge = (props: XChainBridgeProps) => {
     </>
   )
 }
+
+XChainBridge.defaultProps = {
```

Comment:
`defaultProps` is not needed in a tsx component.

Reply:
eslint yelled at me about it: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/require-default-props.md

Reply:
Try exporting `XChainBridgeProps`.